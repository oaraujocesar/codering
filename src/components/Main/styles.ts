import styled from 'styled-components'

export const Wrapper = styled.main`
  display: flex;
  text-align: center;
  align-items: center;
  flex-direction: column;
  justify-content: center;

  width: 100%;
  height: 100%;
  padding: 3rem;

  color: #ffffff;
  background-color: ${(props) => props.theme.backgroundColor};
`
export const Logo = styled.img`
  width: 25rem;
  margin-bottom: 2rem;
`

export const Title = styled.h1`
  font-size: 2.5rem;
`

export const Description = styled.h2`
  font-size: 2rem;
  font-weight: 400;
`

export const Illustration = styled.img`
  margin: 3rem;
  width: min(30rem, 100%);
`
