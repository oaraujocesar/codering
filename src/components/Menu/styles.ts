import styled from 'styled-components'

export const Wrapper = styled.aside`
  top: 0;
  left: 0;
  z-index: 20;
  height: 100vh;
  padding: 40px 0;
  position: sticky;
  background: ${(props) => props.theme.menu};
  box-shadow: 4px 4px 25px rgba(0, 0, 0, 0.16);

  svg.logo {
    padding: 0 39.5px;
    margin-bottom: 50px;
  }
`

export const LogoLink = styled.a`
  cursor: pointer;
  &:hover {
    opacity: 0.8;
  }
`

export const SocialMedia = styled.div`
  margin-top: 24.6rem;
  text-align: center;

  p {
    font-size: 2.2rem;
    line-height: 1.7rem;
    color: ${(props) => props.theme.menuItem};
  }
`

export const SocialList = styled.ul`
  display: flex;
  align-items: center;
  justify-content: center;
  margin-top: 2.4rem;

  li {
    width: 32px;
    height: 32px;
    margin: 0 6px;
  }
`

export const SocialLink = styled.a`
  cursor: pointer;
  height: 100%;
  display: block;

  &:hover {
    svg path:first-child,
    svg path.github {
      fill: ${(props) => props.theme.primaryColor};
    }
  }
`
