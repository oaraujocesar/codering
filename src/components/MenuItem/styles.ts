import styled, { css } from 'styled-components'

const complexMixin = css`
  background: ${(props) => props.theme.menuBackground};
  &::before {
    opacity: 1;
  }
`

export const Wrapper = styled.li<{ isActive: boolean }>`
  width: 100%;
  display: flex;
  position: relative;
  min-height: 80px;

  margin-bottom: 8px;
  border-top: 1.5px solid ${(props) => props.theme.separators};
  border-bottom: 1.5px solid ${(props) => props.theme.separators};

  &:hover {
    background: ${(props) => props.theme.menuBackground};
    &::before {
      opacity: 1;
    }
  }

  &::before {
    top: -2px;
    opacity: 0;
    content: '';
    width: 10px;
    height: 80px;
    background: ${(props) => props.theme.primaryColor};
    position: absolute;
  }

  ${(props) => (props.isActive ? complexMixin : '')}

  a {
    width: 100%;
    display: inherit;
    padding: 18px 50px;
    align-items: center;

    font-size: 2.4rem;
    line-height: 1.7rem;
    color: ${(props) => props.theme.menuItem};

    img {
      margin-right: 20px;
    }
  }
`
