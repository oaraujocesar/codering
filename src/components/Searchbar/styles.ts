import styled from 'styled-components'

export const Wrapper = styled.div`
  position: relative;
  width: 100%;
  max-width: 763px;
`

export const Input = styled.input<{ autocomplete: string }>`
  width: 100%;
  outline: none;
  padding: 9px 28px;
  border-radius: 20px;
  border: 1px solid transparent;
  color: ${(props) => props.theme.textColorGrey};
  background: ${(props) => props.theme.shadeOne};
  border: 2px solid transparent;

  &:focus {
    border: 2px solid ${(props) => props.theme.primaryColor};
  }
`

export const Button = styled.button`
  top: 0;
  right: 0;
  border: none;
  height: 100%;
  outline: none;
  display: flex;
  cursor: pointer;
  padding: 4px 20px;
  position: absolute;
  align-items: center;
  border-radius: 0 20px 20px 0;
  background: ${(props) => props.theme.primaryColor};

  &:hover {
    filter: brightness(0.9);
  }
`
