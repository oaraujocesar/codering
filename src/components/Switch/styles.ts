import styled from 'styled-components'

export const Wrapper = styled.div`
  width: 127px;
  display: flex;
  padding: 4px 11px;
  position: relative;
  border-radius: 20px;
  justify-content: center;
  border: 1px solid transparent;
  background-color: ${(props) => props.theme.shadeOne};
`

export const Handler = styled.div<{ checked: boolean }>`
  top: 3px;
  left: 8px;
  z-index: 1;
  width: 45%;
  position: absolute;
  border-radius: 20px;
  background-color: white;
  height: calc(100% - 6px);

  transition: transform 0.3s ease;
  transform: translateX(${(props) => (props.checked ? '0' : '91%')});
`

export const IconWrapper = styled.div`
  z-index: 2;
  width: 100%;
  height: 100%;
  display: flex;
  cursor: pointer;
  justify-content: center;
`
