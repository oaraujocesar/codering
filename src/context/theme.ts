import { createContext } from 'react'

interface IThemeContextProps {
  dispatch: ({ type }: { type: string; value: string }) => void
}

export const ThemeContext = createContext({} as IThemeContextProps)
