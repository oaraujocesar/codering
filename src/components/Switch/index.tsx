import { useState } from 'react'

import SunIcon from '../Icons/Sun'
import MoonIcon from '../Icons/Moon'

import * as S from './styles'

interface IProps {
  action?(theme: string): void
}

const Switch = ({ action }: IProps) => {
  const [checked, setChecked] = useState(true)

  const toggleSwitch = (position: boolean) => setChecked(position)

  return (
    <S.Wrapper>
      <S.IconWrapper
        role="option"
        onClick={() => {
          toggleSwitch(true)
          action && action('dark')
        }}
      >
        <MoonIcon fill={checked ? '#D6AD45' : '#707070'} />
      </S.IconWrapper>

      <S.IconWrapper
        role="option"
        onClick={() => {
          toggleSwitch(false)
          action && action('light')
        }}
      >
        <SunIcon fill={checked ? '#707070' : '#D6AD45'} />
      </S.IconWrapper>

      <S.Handler role="theme-indicator" checked={checked} />
    </S.Wrapper>
  )
}

export default Switch
