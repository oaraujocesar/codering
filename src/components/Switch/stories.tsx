import { Story, Meta } from '@storybook/react/types-6-0'
import { theme } from '../../styles/theme'
import { ThemeProvider } from 'styled-components'

import Switch from '.'

export default {
  title: 'Theme Switch',
  component: Switch,
  argTypes: {
    currentTheme: { control: { type: 'radio', options: ['light', 'dark'] } }
  }
} as Meta

export const Basic: Story = (args) => {
  const currentTheme = args.currentTheme === 'dark' ? theme.dark : theme.light

  return (
    <ThemeProvider theme={currentTheme}>
      <Switch {...args} />
    </ThemeProvider>
  )
}
