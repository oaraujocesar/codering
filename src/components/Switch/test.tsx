import { render, screen } from '@testing-library/react'

import { theme } from '../../styles/theme'
import { ThemeProvider } from 'styled-components'

import Switch from '.'

describe('<Switch />', () => {
  it('Should render dark colors correctly', async () => {
    const { container } = render(
      <ThemeProvider theme={theme.dark}>
        <Switch />
      </ThemeProvider>
    )

    expect(container.firstElementChild).toHaveStyleRule(
      'background-color',
      theme.dark.shadeOne
    )
    expect(container.firstElementChild).toMatchSnapshot()
  })

  it('Should render light colors correctly', () => {
    const { container } = render(
      <ThemeProvider theme={theme.light}>
        <Switch />
      </ThemeProvider>
    )

    expect(container.firstElementChild).toHaveStyleRule(
      'background-color',
      theme.light.shadeOne
    )
    expect(container.firstElementChild).toMatchSnapshot()
  })

  it('Should render two options', () => {
    render(
      <ThemeProvider theme={theme.dark}>
        <Switch />
      </ThemeProvider>
    )

    expect(screen.queryAllByRole('option')).toHaveLength(2)
  })
})
