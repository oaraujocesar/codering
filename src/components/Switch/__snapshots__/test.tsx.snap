// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Switch /> Should render dark colors correctly 1`] = `
.c0 {
  width: 127px;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  padding: 4px 11px;
  position: relative;
  border-radius: 20px;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  border: 1px solid transparent;
  background-color: #424242;
}

.c2 {
  top: 3px;
  left: 8px;
  z-index: 1;
  width: 45%;
  position: absolute;
  border-radius: 20px;
  background-color: white;
  height: calc(100% - 6px);
  -webkit-transition: -webkit-transform 0.3s ease;
  -webkit-transition: transform 0.3s ease;
  transition: transform 0.3s ease;
  -webkit-transform: translateX(0);
  -ms-transform: translateX(0);
  transform: translateX(0);
}

.c1 {
  z-index: 2;
  width: 100%;
  height: 100%;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  cursor: pointer;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
}

<div
  class="c0"
>
  <div
    class="c1"
    role="option"
  >
    <svg
      class=""
      fill="none"
      height="26"
      viewBox="0 0 26 26"
      width="26"
      xmlns="http://www.w3.org/2000/svg"
    >
      <path
        d="M12 13.5C9.12808 10.627 9.42505 5.69941 10.8865 2.16666C8.85522 2.56749 6.91497 3.55441 5.3398 5.12957C1.10939 9.35999 1.10939 16.2197 5.3398 20.4501C9.5713 24.6816 16.4299 24.6805 20.6614 20.4501C22.2366 18.8749 23.2224 16.9357 23.6243 14.9045C20.0905 16.3648 14.8719 16.3719 12 13.5Z"
        fill="#D6AD45"
      />
    </svg>
  </div>
  <div
    class="c1"
    role="option"
  >
    <svg
      class=""
      fill="none"
      height="26"
      viewBox="0 0 26 26"
      width="26"
      xmlns="http://www.w3.org/2000/svg"
    >
      <path
        clip-rule="evenodd"
        d="M13 0C13.431 0 13.8443 0.171205 14.149 0.475951C14.4538 0.780698 14.625 1.19402 14.625 1.625V3.25C14.625 3.68098 14.4538 4.0943 14.149 4.39905C13.8443 4.70379 13.431 4.875 13 4.875C12.569 4.875 12.1557 4.70379 11.851 4.39905C11.5462 4.0943 11.375 3.68098 11.375 3.25V1.625C11.375 1.19402 11.5462 0.780698 11.851 0.475951C12.1557 0.171205 12.569 0 13 0ZM19.5 13C19.5 14.7239 18.8152 16.3772 17.5962 17.5962C16.3772 18.8152 14.7239 19.5 13 19.5C11.2761 19.5 9.62279 18.8152 8.4038 17.5962C7.18482 16.3772 6.5 14.7239 6.5 13C6.5 11.2761 7.18482 9.62279 8.4038 8.4038C9.62279 7.18482 11.2761 6.5 13 6.5C14.7239 6.5 16.3772 7.18482 17.5962 8.4038C18.8152 9.62279 19.5 11.2761 19.5 13ZM18.746 21.0437L19.8949 22.1926C20.2014 22.4886 20.6118 22.6524 21.0379 22.6487C21.464 22.645 21.8715 22.4741 22.1728 22.1728C22.4741 21.8715 22.645 21.464 22.6487 21.0379C22.6524 20.6118 22.4886 20.2013 22.1926 19.8949L21.0437 18.746C20.7373 18.45 20.3268 18.2862 19.9007 18.2899C19.4747 18.2936 19.0671 18.4645 18.7658 18.7658C18.4645 19.0671 18.2936 19.4746 18.2899 19.9007C18.2862 20.3268 18.45 20.7373 18.746 21.0437ZM22.191 3.80737C22.4956 4.11211 22.6668 4.52536 22.6668 4.95625C22.6668 5.38714 22.4956 5.80039 22.191 6.10512L21.0437 7.254C20.8938 7.4092 20.7145 7.533 20.5163 7.61816C20.318 7.70333 20.1048 7.74816 19.889 7.75003C19.6733 7.7519 19.4593 7.71079 19.2596 7.62908C19.0599 7.54738 18.8784 7.42672 18.7259 7.27414C18.5733 7.12156 18.4526 6.94013 18.3709 6.74042C18.2892 6.54072 18.2481 6.32674 18.25 6.11097C18.2518 5.8952 18.2967 5.68197 18.3818 5.48372C18.467 5.28546 18.5908 5.10615 18.746 4.95625L19.8949 3.80737C20.1996 3.50273 20.6129 3.3316 21.0437 3.3316C21.4746 3.3316 21.8879 3.50273 22.1926 3.80737H22.191ZM24.375 14.625C24.806 14.625 25.2193 14.4538 25.524 14.149C25.8288 13.8443 26 13.431 26 13C26 12.569 25.8288 12.1557 25.524 11.8509C25.2193 11.5462 24.806 11.375 24.375 11.375H22.75C22.319 11.375 21.9057 11.5462 21.601 11.8509C21.2962 12.1557 21.125 12.569 21.125 13C21.125 13.431 21.2962 13.8443 21.601 14.149C21.9057 14.4538 22.319 14.625 22.75 14.625H24.375ZM13 21.125C13.431 21.125 13.8443 21.2962 14.149 21.6009C14.4538 21.9057 14.625 22.319 14.625 22.75V24.375C14.625 24.806 14.4538 25.2193 14.149 25.524C13.8443 25.8288 13.431 26 13 26C12.569 26 12.1557 25.8288 11.851 25.524C11.5462 25.2193 11.375 24.806 11.375 24.375V22.75C11.375 22.319 11.5462 21.9057 11.851 21.6009C12.1557 21.2962 12.569 21.125 13 21.125ZM4.95625 7.254C5.10712 7.40498 5.28626 7.52476 5.48342 7.60651C5.68059 7.68826 5.89192 7.73038 6.10536 7.73045C6.3188 7.73053 6.53017 7.68856 6.72739 7.60695C6.92462 7.52534 7.10383 7.40568 7.25481 7.25481C7.40579 7.10394 7.52558 6.92481 7.60733 6.72764C7.68908 6.53048 7.73119 6.31914 7.73127 6.1057C7.73134 5.89226 7.68938 5.68089 7.60777 5.48367C7.52615 5.28645 7.4065 5.10723 7.25562 4.95625L6.10512 3.80737C5.79865 3.51137 5.38817 3.34758 4.9621 3.35128C4.53603 3.35498 4.12846 3.52588 3.82717 3.82717C3.52588 4.12846 3.35498 4.53603 3.35128 4.9621C3.34758 5.38817 3.51137 5.79865 3.80737 6.10512L4.95625 7.254ZM7.254 21.0437L6.10512 22.1926C5.79865 22.4886 5.38817 22.6524 4.9621 22.6487C4.53603 22.645 4.12846 22.4741 3.82717 22.1728C3.52588 21.8715 3.35498 21.464 3.35128 21.0379C3.34758 20.6118 3.51137 20.2013 3.80737 19.8949L4.95625 18.746C5.26273 18.45 5.67321 18.2862 6.09928 18.2899C6.52535 18.2936 6.93292 18.4645 7.2342 18.7658C7.53549 19.0671 7.70639 19.4746 7.71009 19.9007C7.7138 20.3268 7.55001 20.7373 7.254 21.0437ZM3.25 14.625C3.68098 14.625 4.0943 14.4538 4.39905 14.149C4.70379 13.8443 4.875 13.431 4.875 13C4.875 12.569 4.70379 12.1557 4.39905 11.8509C4.0943 11.5462 3.68098 11.375 3.25 11.375H1.625C1.19402 11.375 0.780698 11.5462 0.475951 11.8509C0.171205 12.1557 0 12.569 0 13C0 13.431 0.171205 13.8443 0.475951 14.149C0.780698 14.4538 1.19402 14.625 1.625 14.625H3.25Z"
        fill="#707070"
        fill-rule="evenodd"
      />
    </svg>
  </div>
  <div
    class="c2"
    role="theme-indicator"
  />
</div>
`;

exports[`<Switch /> Should render light colors correctly 1`] = `
.c0 {
  width: 127px;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  padding: 4px 11px;
  position: relative;
  border-radius: 20px;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  border: 1px solid transparent;
  background-color: #E6E6E6;
}

.c2 {
  top: 3px;
  left: 8px;
  z-index: 1;
  width: 45%;
  position: absolute;
  border-radius: 20px;
  background-color: white;
  height: calc(100% - 6px);
  -webkit-transition: -webkit-transform 0.3s ease;
  -webkit-transition: transform 0.3s ease;
  transition: transform 0.3s ease;
  -webkit-transform: translateX(0);
  -ms-transform: translateX(0);
  transform: translateX(0);
}

.c1 {
  z-index: 2;
  width: 100%;
  height: 100%;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  cursor: pointer;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
}

<div
  class="c0"
>
  <div
    class="c1"
    role="option"
  >
    <svg
      class=""
      fill="none"
      height="26"
      viewBox="0 0 26 26"
      width="26"
      xmlns="http://www.w3.org/2000/svg"
    >
      <path
        d="M12 13.5C9.12808 10.627 9.42505 5.69941 10.8865 2.16666C8.85522 2.56749 6.91497 3.55441 5.3398 5.12957C1.10939 9.35999 1.10939 16.2197 5.3398 20.4501C9.5713 24.6816 16.4299 24.6805 20.6614 20.4501C22.2366 18.8749 23.2224 16.9357 23.6243 14.9045C20.0905 16.3648 14.8719 16.3719 12 13.5Z"
        fill="#D6AD45"
      />
    </svg>
  </div>
  <div
    class="c1"
    role="option"
  >
    <svg
      class=""
      fill="none"
      height="26"
      viewBox="0 0 26 26"
      width="26"
      xmlns="http://www.w3.org/2000/svg"
    >
      <path
        clip-rule="evenodd"
        d="M13 0C13.431 0 13.8443 0.171205 14.149 0.475951C14.4538 0.780698 14.625 1.19402 14.625 1.625V3.25C14.625 3.68098 14.4538 4.0943 14.149 4.39905C13.8443 4.70379 13.431 4.875 13 4.875C12.569 4.875 12.1557 4.70379 11.851 4.39905C11.5462 4.0943 11.375 3.68098 11.375 3.25V1.625C11.375 1.19402 11.5462 0.780698 11.851 0.475951C12.1557 0.171205 12.569 0 13 0ZM19.5 13C19.5 14.7239 18.8152 16.3772 17.5962 17.5962C16.3772 18.8152 14.7239 19.5 13 19.5C11.2761 19.5 9.62279 18.8152 8.4038 17.5962C7.18482 16.3772 6.5 14.7239 6.5 13C6.5 11.2761 7.18482 9.62279 8.4038 8.4038C9.62279 7.18482 11.2761 6.5 13 6.5C14.7239 6.5 16.3772 7.18482 17.5962 8.4038C18.8152 9.62279 19.5 11.2761 19.5 13ZM18.746 21.0437L19.8949 22.1926C20.2014 22.4886 20.6118 22.6524 21.0379 22.6487C21.464 22.645 21.8715 22.4741 22.1728 22.1728C22.4741 21.8715 22.645 21.464 22.6487 21.0379C22.6524 20.6118 22.4886 20.2013 22.1926 19.8949L21.0437 18.746C20.7373 18.45 20.3268 18.2862 19.9007 18.2899C19.4747 18.2936 19.0671 18.4645 18.7658 18.7658C18.4645 19.0671 18.2936 19.4746 18.2899 19.9007C18.2862 20.3268 18.45 20.7373 18.746 21.0437ZM22.191 3.80737C22.4956 4.11211 22.6668 4.52536 22.6668 4.95625C22.6668 5.38714 22.4956 5.80039 22.191 6.10512L21.0437 7.254C20.8938 7.4092 20.7145 7.533 20.5163 7.61816C20.318 7.70333 20.1048 7.74816 19.889 7.75003C19.6733 7.7519 19.4593 7.71079 19.2596 7.62908C19.0599 7.54738 18.8784 7.42672 18.7259 7.27414C18.5733 7.12156 18.4526 6.94013 18.3709 6.74042C18.2892 6.54072 18.2481 6.32674 18.25 6.11097C18.2518 5.8952 18.2967 5.68197 18.3818 5.48372C18.467 5.28546 18.5908 5.10615 18.746 4.95625L19.8949 3.80737C20.1996 3.50273 20.6129 3.3316 21.0437 3.3316C21.4746 3.3316 21.8879 3.50273 22.1926 3.80737H22.191ZM24.375 14.625C24.806 14.625 25.2193 14.4538 25.524 14.149C25.8288 13.8443 26 13.431 26 13C26 12.569 25.8288 12.1557 25.524 11.8509C25.2193 11.5462 24.806 11.375 24.375 11.375H22.75C22.319 11.375 21.9057 11.5462 21.601 11.8509C21.2962 12.1557 21.125 12.569 21.125 13C21.125 13.431 21.2962 13.8443 21.601 14.149C21.9057 14.4538 22.319 14.625 22.75 14.625H24.375ZM13 21.125C13.431 21.125 13.8443 21.2962 14.149 21.6009C14.4538 21.9057 14.625 22.319 14.625 22.75V24.375C14.625 24.806 14.4538 25.2193 14.149 25.524C13.8443 25.8288 13.431 26 13 26C12.569 26 12.1557 25.8288 11.851 25.524C11.5462 25.2193 11.375 24.806 11.375 24.375V22.75C11.375 22.319 11.5462 21.9057 11.851 21.6009C12.1557 21.2962 12.569 21.125 13 21.125ZM4.95625 7.254C5.10712 7.40498 5.28626 7.52476 5.48342 7.60651C5.68059 7.68826 5.89192 7.73038 6.10536 7.73045C6.3188 7.73053 6.53017 7.68856 6.72739 7.60695C6.92462 7.52534 7.10383 7.40568 7.25481 7.25481C7.40579 7.10394 7.52558 6.92481 7.60733 6.72764C7.68908 6.53048 7.73119 6.31914 7.73127 6.1057C7.73134 5.89226 7.68938 5.68089 7.60777 5.48367C7.52615 5.28645 7.4065 5.10723 7.25562 4.95625L6.10512 3.80737C5.79865 3.51137 5.38817 3.34758 4.9621 3.35128C4.53603 3.35498 4.12846 3.52588 3.82717 3.82717C3.52588 4.12846 3.35498 4.53603 3.35128 4.9621C3.34758 5.38817 3.51137 5.79865 3.80737 6.10512L4.95625 7.254ZM7.254 21.0437L6.10512 22.1926C5.79865 22.4886 5.38817 22.6524 4.9621 22.6487C4.53603 22.645 4.12846 22.4741 3.82717 22.1728C3.52588 21.8715 3.35498 21.464 3.35128 21.0379C3.34758 20.6118 3.51137 20.2013 3.80737 19.8949L4.95625 18.746C5.26273 18.45 5.67321 18.2862 6.09928 18.2899C6.52535 18.2936 6.93292 18.4645 7.2342 18.7658C7.53549 19.0671 7.70639 19.4746 7.71009 19.9007C7.7138 20.3268 7.55001 20.7373 7.254 21.0437ZM3.25 14.625C3.68098 14.625 4.0943 14.4538 4.39905 14.149C4.70379 13.8443 4.875 13.431 4.875 13C4.875 12.569 4.70379 12.1557 4.39905 11.8509C4.0943 11.5462 3.68098 11.375 3.25 11.375H1.625C1.19402 11.375 0.780698 11.5462 0.475951 11.8509C0.171205 12.1557 0 12.569 0 13C0 13.431 0.171205 13.8443 0.475951 14.149C0.780698 14.4538 1.19402 14.625 1.625 14.625H3.25Z"
        fill="#707070"
        fill-rule="evenodd"
      />
    </svg>
  </div>
  <div
    class="c2"
    role="theme-indicator"
  />
</div>
`;
